README.md

```diff
@@ -51,6 +50,7 @@ Run the client specifying an email address you want to associate the data with
 | `IMUP_REALTIME_AUTHORIZED`         | imup API address for real-time authorized | `"https://api.imup.io/v1/auth/real-timeAuthorized"`          |
 | `IMUP_REALTIME_CONFIG`             | imup API address for reloadable config    | `"https://api.imup.io/v1/realtime/config"`                   |
 | `IMUP_DATA_LENGTH`                 | imup data length per interval             | `"15"`                                                       |
+| `LOG_TO_FILE`                      | log output will write to a file           | `"the root default directory for user-specific cached data"` |
```

Comment:
the default here feels confusing. maybe if we just remove the string around it that'd do it for me so its clear its not literally that phrase but I still wouldn't really know what it meant.

Reply:
i just realized this is supposed to be a boolean. this should be true/false for the default I think and move your string into the description?